cornichon-kafka/src/main/scala/com/github/agourlay/cornichon/kafka/KafkaDsl.scala

```diff
@@ -0,0 +1,126 @@
+package com.github.agourlay.cornichon.kafka
+
+import com.github.agourlay.cornichon.core.Session
+import com.github.agourlay.cornichon.dsl.Dsl
+import com.github.agourlay.cornichon.feature.BaseFeature
+import com.github.agourlay.cornichon.json.CornichonJson
+import com.github.agourlay.cornichon.steps.regular.EffectStep
+import com.typesafe.config.ConfigFactory
+import org.apache.kafka.clients.producer._
+import org.apache.kafka.common.serialization.{ StringDeserializer, StringSerializer }
+import com.github.agourlay.cornichon.kafka.KafkaDsl._
+import org.apache.kafka.clients.consumer.{ ConsumerConfig, ConsumerRecord, KafkaConsumer }
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.{ Future, Promise }
+
+trait KafkaDsl {
+  this: BaseFeature with Dsl ⇒
+
+  override private[cornichon] lazy val config = BaseFeature.config.copy(executeScenariosInParallel = false)
```

Comment:
I think this constraint should be mentioned in the doc.

Reply:
I think it is mentioned in the doc, but maybe its not explicetly mentioned enough. I will bold it

Reply:
boldened